File: archive/java/resonantinduction/mechanical/ClientProxy.java
package resonantinduction.mechanical;

import resonantinduction.mechanical.render.MechanicalBlockRenderingHandler;
import cpw.mods.fml.client.registry.RenderingRegistry;

public class ClientProxy extends CommonProxy
{
	public void preInit()
	{
		RenderingRegistry.registerBlockHandler(MechanicalBlockRenderingHandler.INSTANCE);
	}
}

File: src/main/scala/resonantinduction/archaic/ClientProxy.scala
package resonantinduction.archaic

import cpw.mods.fml.client.registry.ClientRegistry
import resonantinduction.archaic.firebox.{RenderHotPlate, TileHotPlate}
import resonantinduction.archaic.process.{RenderCastingMold, RenderMillstone, TileCastingMold, TileMillstone}

class ClientProxy extends CommonProxy
{
  override def init
  {
    ClientRegistry.bindTileEntitySpecialRenderer(classOf[TileCastingMold], new RenderCastingMold)
    ClientRegistry.bindTileEntitySpecialRenderer(classOf[TileMillstone], new RenderMillstone)
    ClientRegistry.bindTileEntitySpecialRenderer(classOf[TileHotPlate], new RenderHotPlate)
  }
}